app/helpers/admin/payroll_run_helper.rb

```diff
@@ -0,0 +1,13 @@
+module Admin
+  module PayrollRunHelper
+    def next_payroll_file_to_cantium_due_date(current_date = Date.today)
+      all_fridays_next_month(current_date)[-3]
+    end
+
+    private
+
+    def all_fridays_next_month(current_date)
+      current_date.next_month.beginning_of_month.step(current_date.next_month.end_of_month).select { |day| day.friday? }
```

Comment:
I always think `(start_date..end_date)` is more expressive than using `step`. Might be a bit more readable to store next month in a variable too I reckon.

```ruby
next_month = current_date.next_month
(next_month.beginning_of_month..next_month.end_of_month).select { |day| day.friday? }
```

Reply:
Nice, these are the ruby things I like to learn, thank you!